docs/src/meshes/MeshDataMeshes.md

```diff
@@ -0,0 +1,70 @@
+* variables ending in `q` (e.g., `rq, sq,...` and `xq, yq,...`) correspond to values at volume quadrature points. 
+* variables ending in `f` (e.g., `rf, sf,...` and `xf, yf,...`) correspond to values at face quadrature points. 
+* variables ending in `p` (e.g., `rp, sp,...`)
+* `V` matrices correspond to interpolation matrices from nodal interpolation points, e.g., `Vq` interpolates to volume quadrature points, `Vf` interpolates to face quadrature points. 
+* geometric quantities in `MeshData` are stored as matrices of dimension ``\text{number of points per element } \times \text{number of elements}``. 
+
+Quantities in `rd::RefElemData`: 
+* `rd.Np, rd.Nq, rd.Nf`: the number of nodal interpolation points, volume quadrature points, and face quadrature points on the reference element, respectively. 
+* `rd.Vq`: interpolation matrices from values at nodal interpolation points to volume quadrature points
+* `rd.wq`: volume quadrature weights on the reference element
+* `rd.Vf`: interpolation matrices from values at nodal interpolation points to face quadrature points
+* `rd.wf`: a vector containing face quadrature weights on the reference element
+* `rd.M`: the quadrature-based mass matrix, computed via `rd.Vq' * diagm(rd.wq) * rd.Vq`.
+* `rd.Pq`: a quadrature-based ``L^2`` projection matrix `rd.Pq = rd.M \ rd.Vq' * diagm(rd.wq)` which maps between values at quadrature points and values at nodal points. 
+* `Dr, Ds, Dt` matrices are nodal differentiation matrices with respect to the ``r,s,t`` coordinates, e.g., `Dr*f.(r,s)` approximates the derivative of ``f(r,s)`` at nodal points. 
+
+Quantities in `md::MeshData`: 
+* `md.xyz` is a tuple of matrices `md.x`, `md.y`, `md.z`, where column `e` contains coordinates of physical interpolation points. 
+* `md.xyzq` is a tuple of matrices `md.xq`, `md.yq`, `md.zq`, where column `e` contains coordinates of physical quadrature points. 
+* `md.rxJ, md.sxJ, ...` are matrices where column `e` contains values of ``J\frac{\partial r}{\partial x}``, ``J\frac{\partial s}{\partial x}``, etc. at nodal interpolation points on the element `e`.
+* `md.J` is a matrix where column `e` contains values of ``J`` at nodal interpolation points.
+* `md.sJ` is a matrix where column `e` contains values of the face Jacobian (e.g., determinant of the geometric mapping between a physical face and a reference face) at face quadrature points.
+* `md.nxJ, md.nyJ, ...` are matrices where column `e` contains values of components of the unit normal scaled by the face Jacobian `md.sJ` at face quadrature points.
+
+For more details, please see the [StartUpDG.jl docs](https://jlchan.github.io/StartUpDG.jl/dev/). 
+## Special options
+
+Trixi solvers on simplicial meshes use `DG` solver types with the basis field set as a `RefElemData` type. By default, `RefElemData(Tri(), N)` will generate data for a degree ``N`` polynomial approximation on a triangle. There are also several parameters which can be tweaked:
```

Comment:
```suggestion
Trixi solvers on simplicial meshes use `DG` solver types with the `basis` field set as a `RefElemData` type. By default, `RefElemData(Tri(), N)` will generate data for a degree ``N`` polynomial approximation on a triangle. There are also several parameters which can be tweaked:
```

How strong do you prefer to use `N` as variable name for the polynomial degree? It might make sense to use `polydeg` (here and in the elixirs) as we do in other parts of Trixi. 
Background: We had quite a discussion about this in the past since some core developers grew up using `N` for the polynomial degree, too. In the end, we decided to use `polydeg` since that makes it easier to parse for people from other schools. 
Anyways, we need to find a balance between Trixi and StartUpDG, so I would not insist on a specific variable name here.

Reply:
I'm happy to use polydeg within any Trixi elixirs. 

I'm less sure about changing 'N' in 'RefElemData' though. What do you think about overloading 'getproperty' so that 'rd.polydeg' returns N, but not making that property public?

Reply:
Sounds like an option. On the other hand, I think it's fine to query the polynomial degree using the function `polydeg` inside Trixi (if necessary at all). I was more thinking about the argument name used to set up everything.

Reply:
So 'RefElemData(Tri(), polydeg)' in the elixir setup? That sounds great to me